package/kernel/mac80211/patches/build/236-fix-genlmsg_multicast_allns-build-error-on-kernel-6.6.59.patch

```diff
@@ -0,0 +1,100 @@
+--- a/backport-include/net/genetlink.h
++++ b/backport-include/net/genetlink.h
+@@ -150,7 +150,7 @@ int genlmsg_multicast(const struct genl_
```

Comment:
https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit?h=v6.11.6&id=e0f83d268974dab0361d11904dfc9acec53f96a6

Please backport the full patch and keep the header.

Reply:
The patch cannot be fully backported because it affects more files. Only changes related to files in mac80211 are imported.